src/angularfire2.ts

```diff
@@ -23,48 +25,49 @@ export class AngularFire {
   list: (url:string, opts?:FirebaseListFactoryOpts) => FirebaseListObservable<any[]>;
   object: (url: string, opts?:FirebaseObjectFactoryOpts) => FirebaseObjectObservable<any>;
   constructor(
-    @Inject(FirebaseUrl) private fbUrl:string,
-    public auth:FirebaseAuth,
+    @Inject(FirebaseConfig) private fbUrl:string,
```

Comment:
I created an issue (#245) before noticing that this PR was still open. The injected `FirebaseConfig` parameter is a `FirebaseAppConfig` instance - not a `string`. Also, the `list` and `object` members aren't used - `list` and `object` are functions of the database mamber.